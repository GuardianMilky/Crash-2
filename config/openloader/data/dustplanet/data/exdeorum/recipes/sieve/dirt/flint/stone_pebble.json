{"type":"exdeorum:sieve","ingredient":{"item":"minecraft:dirt"},"mesh":"exdeorum:flint_mesh","result":"exdeorum:stone_pebble","result_amount":{"type":"minecraft:binomial","n":7.0,"p":0.6}}