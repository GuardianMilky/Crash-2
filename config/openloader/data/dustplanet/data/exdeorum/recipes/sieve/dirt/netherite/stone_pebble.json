{"type":"exdeorum:sieve","ingredient":{"item":"minecraft:dirt"},"mesh":"exdeorum:netherite_mesh","result":"exdeorum:stone_pebble","result_amount":{"type":"minecraft:binomial","n":5.0,"p":0.4}}