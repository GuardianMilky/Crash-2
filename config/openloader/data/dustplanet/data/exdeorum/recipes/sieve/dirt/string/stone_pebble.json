{"type":"exdeorum:sieve","ingredient":{"item":"minecraft:dirt"},"mesh":"exdeorum:string_mesh","result":"exdeorum:stone_pebble","result_amount":{"type":"minecraft:binomial","n":7.0,"p":0.6}}