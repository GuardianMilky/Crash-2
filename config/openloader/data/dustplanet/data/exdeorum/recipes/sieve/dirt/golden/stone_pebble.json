{"type":"exdeorum:sieve","ingredient":{"item":"minecraft:dirt"},"mesh":"exdeorum:golden_mesh","result":"exdeorum:stone_pebble","result_amount":{"type":"minecraft:binomial","n":8.0,"p":0.7}}